src/service.rs

```diff
@@ -136,7 +175,11 @@ impl S3Service {
             Err(ref err) => error!(%err),
         };
 
-        Ok(ret?)
+        let mut res = ret?;
+        let _ = res.headers_mut().insert(ACCESS_CONTROL_ALLOW_ORIGIN, HeaderValue::from_static("*"));
```

Comment:
Is this a good idea, considering we're serving the frontend with this code and they should always be on the same domain?

Reply:
same as above.